src/ajax/xhr.js

```diff
@@ -15,156 +14,109 @@ jQuery.ajaxSettings.xhr = function() {
 var xhrSuccessStatus = {
 
 		// File protocol always yields status code 0, assume 200
-		0: 200,
-
-		// Support: IE <=9 only
-		// #1450: sometimes IE returns 1223 when it should be 204
-		1223: 204
-	},
-	xhrSupported = jQuery.ajaxSettings.xhr();
-
-support.cors = !!xhrSupported && ( "withCredentials" in xhrSupported );
```

Comment:
The scenarios where I've ever seen this used are `jQuery.support.cors = true`, that is, just setting the value. I wonder if there are a lot of cases where people check to see if it's `true` in their own code? If so it might be safest to set it to `true` here. Here's an example of some code that would break if we didn't: https://github.com/victorquinn/Backbone.CrossDomain/issues/1

Reply:
I guess we could add a line with:
```
jQuery.support.cors = true;
```
to `deprecated.js` and add a warning to Migrate for code accessing it?

Reply:
It's mainly useful of you want to gracefully degrade and/or fail in browsers without cors which will be none with the new support list. I think it would be better to keep it set to true for a good while in `deprecated.js` before removing it.

Reply:
We could also have Migrate fill in those properties as it's easy to do from outside. Technically speaking, external access to `jQuery.support` has never been supported.